packages/dapp/src/components/Navbar/Menu/Menus.tsx

```diff
@@ -1,26 +1,26 @@
+import { useMenuStore } from '@transferto/dapp/src/stores/menu';
 import { SyntheticEvent } from 'react';
-import { ChainsMenu, ConnectedMenu, MainMenu, WalletSelectMenu } from '.';
-import { useMenu } from '../../../providers/MenuProvider';
+import { shallow } from 'zustand/shallow';
+import { ChainsMenu, MainMenu, WalletMenu, WalletSelectMenu } from '.';
+import { SupportModal } from '../../SupportModal';
 
 export const Menus = () => {
-  const menu = useMenu();
+  const [anchorRef] = useMenuStore((state) => [state.anchorRef], shallow);
```

Comment:
When we have one property we can use it without shallow comparison like this

`const anchorRef = useMenuStore((state) => state.anchorRef);`

Reply:
Yes, that looks shorter, cleaner and is the way to go! Good find! There is numerous places which now got refactored.